ingestfile

```diff
@@ -58,6 +40,18 @@ _add_mediaid_to_queue(){
     [ -n "${MEDIAID}" ] || { _report -wt "Error: mediaid is undefined in add_media_to_queue function" ; _writeerrorlog "_add_mediaid_to_queue" "The media id is undefined in the _add_mediaid_to_queue function, and caused the script to exit." ; exit 1 ;};
     [ -n "${QUEUEFILE}" ] || { _report -wt "Error: queuefile is undefined in add_media_to_queue function" ; _writeerrorlog "_add_mediaid_to_queue" "The queuefile is undefined in the _add_mediaid_to_queue function, and caused the script to exit." ; exit 1 ;};
     echo "${MEDIAID}" >> "${QUEUEFILE}"
+    NEXT=$(head -n 1 "${QUEUEFILE}")
+    while [ "${MEDIAID}" != "${NEXT}" ] ; do
```

Comment:
before this section was inside a `if [ "${PRIORITY}" = "put in queue" ] ; then` condition, but now the `${PRIORITY}` value appears unconsidered. Can the user still force ingestfile to start regardless of the queue?

Reply:
see line 486 - `_add_mediaid_to_queue` will only run if `[ "${PRIORITY}" = "put in queue" ]` is true